internal/configs/version2/nginx-plus.virtualserver.tmpl

```diff
@@ -232,7 +232,39 @@ server {
         proxy_pass {{ $l.InternalProxyPass }};
         {{ end }}
 
-        {{ if $l.ProxyPass }}
+
+        {{if $l.GrpcPass}}
+        grpc_connect_timeout {{$l.ProxyConnectTimeout}};
+        grpc_read_timeout {{$l.ProxyReadTimeout}};
+        grpc_send_timeout {{$l.ProxySendTimeout}};
+        grpc_set_header Host $host;
+        grpc_set_header X-Real-IP $remote_addr;
+        grpc_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        grpc_set_header X-Forwarded-Host $host;
+        grpc_set_header X-Forwarded-Port $server_port;
+        grpc_set_header X-Forwarded-Proto {{ with $s.TLSRedirect }}{{ .BasedOn }}{{ else }}$scheme{{ end }};
+
+            {{ if $l.ProxyBufferSize}}
+        grpc_buffer_size {{$l.ProxyBufferSize}};
+            {{ end}}
+        grpc_pass {{ $l.GrpcPass }}{{ $l.ProxyPassRewrite }}
```

Comment:
make sure to not forget about `;` at the end of directives. Otherwise, NGINX will fail to reload the config.

Reply:
thanks, fixed